7_react/react2/week1/my-app/src/Who.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+function WhoAreWe(){
+    return(
+    <div>
+    <h2 className='header'>
+    Who we are
+    </h2>
+<p>

```

Comment:
It is always a good idea to format the code before committing. It will also create a good impression in job interviews, code quizzes (for job applications) etc.
See https://code.visualstudio.com/docs/editor/codebasics#_formatting